### ISSUE TITLE
RTX 4060: stuck at 60Hz

### ISSUE THREAD

**Issue by Kimiblock** [NVIDIA/open-gpu-kernel-modules#724]
### NVIDIA Open GPU Kernel Modules Version

560.35.03

### Please confirm this issue does not happen with the proprietary driver (of the same version). This issue tracker is only for bugs specific to the open kernel driver.

- [x] I confirm that this does not happen with the proprietary driver package.

### Operating System and Version

Arch Linux

### Kernel Release

6.11.5-arch1-1

### Please confirm you are running a stable release kernel (e.g. not a -rc). We do not accept bug reports for unreleased kernels.

- [x] I am running on a stable kernel release.

### Hardware: GPU

GPU 0: NVIDIA GeForce RTX 4060 Laptop GPU (UUID: GPU-8329a1c6-f6a8-288a-79e5-65313cf3b82c)

### Describe the bug

After switching from proprietary 560 to open 560 drivers, GNOME no longer shows an option to enable 240Hz output.

![Image](https://github.com/user-attachments/assets/31b5511c-bc97-47e8-b9ed-49c083ce8dd6)

### To Reproduce

```
pacman -S nvidia-open
```

### Bug Incidence

Always

### nvidia-bug-report.log.gz

[nvidia-bug-report.log.gz](https://github.com/user-attachments/files/17555728/nvidia-bug-report.log.gz)

### More Info

![Image](https://github.com/user-attachments/assets/d8cb139f-eed4-491c-941a-da07762a2540)

**Comment by Kimiblock** [comment]
I did a little bit of experiment and it turns out that `NVreg_RegistryDwords="RMForcePstate=0"` was the root cause. But according to [this documentation](https://docs.nvidia.com/gameworks/content/gameworkslibrary/coresdk/nvapi/group__gpupstate.html), P0 should be "best performance"...

**Comment by mtijanic** [comment]
P0 is maximum performance, yes. However, please note that the parameter above is not meant for end user use. It is an internal, test only knob that was never meant to be exposed (albeit trivial to discover by analyzing the driver; it's also not meant to be hidden). I can think of several reasons why it might mess with Is Mode Possible checks.

I really can't recommend using these `RegistryDwords` knobs (especially the undocumented ones) for anything other than temporary tests. We don't QA systems with these set, so you'll likely be hitting more issues where it's unclear what the root cause is.

**Comment by Kimiblock** [comment]
That’s true, but open drivers currently have stuttering issues after idle. I’m looking for ways to mitigate it.--Sincerely, KimiblockOn Oct 29, 2024, at 8:05 PM, Milos Tijanic ***@***.***> wrote:﻿
P0 is maximum performance, yes. However, please note that the parameter above is not meant for end user use. It is an internal, test only knob that was never meant to be exposed (albeit trivial to discover by analyzing the driver; it's also not meant to be hidden). I can think of several reasons why it might mess with Is Mode Possible checks.
I really can't recommend using these RegistryDwords knobs (especially the undocumented ones) for anything other than temporary tests. We don't QA systems with these set, so you'll likely be hitting more issues where it's unclear what the root cause is.

**Comment by Binary-Eater** [comment]
@Kimiblock since this issue was open, we have made some improvements to minimize stutter in the `nvidia-drm` path when GSP is in use. If you are still observing those types of issues, can you open a separate issue?
